Source/WTF/wtf/PlatformHave.h

```diff
@@ -117,59 +117,59 @@
 #endif
 
 /* FIXME: Remove after CMake build enabled on Darwin */
-#if OS(DARWIN)
+#if OS(DARWIN) && !defined(BUILDING_WITH_CMAKE)
```

Comment:
This looks very weird, why does the build system define whether we have `errno.h` in the SDK? 

As the actual reason is to not redeclare, perhaps `!defined()` would be more appropriate.